ui/page/receive_page.go

```diff
@@ -105,7 +106,7 @@ func NewReceivePage(l *load.Load) *ReceivePage {
 				pg.currentAddress = currentAddress
 			}
 
-			pg.generateQRForAddress()
+			pg.generateQRForAddress(currentAddress)
```

Comment:
Please revert this... 
Currently we already have a `pg.currentAddress` i dont see the need to pass parameters around.

`pg,currentAddress` is set before `pg.generateQRForAddress()` is called so no need for he extra `currentAddress` param.

Although i see we may have some challenge here though especially when there is an error with `currentAddress`.

Kindly move `pg.generateQRForAddress()`into the `if else` statement. That way, we only try to generate the QrCode when there is an address available.